src/lib/eap_aka_sim/encode.c

```diff
@@ -347,25 +345,16 @@ static ssize_t encode_value(uint8_t *out, size_t outlen,
 	 */
 	case FR_RES:
 	{
-		uint16_t	res_len = htons(vp->vp_length * 8);	/* Get length in bits */
-
-		uint8_t		*p = out;
-
 		if ((vp->vp_length < 4) || (vp->vp_length > 16)) {
 			fr_strerror_printf("%s: AKA-RES Length must be between 4-16 bytes, got %zu bytes",
 					   __FUNCTION__, vp->vp_length);
 			return PAIR_ENCODE_FATAL_ERROR;
 		}
 
-		CHECK_FREESPACE(outlen, vp->vp_length + 2);
-
-		memcpy(p, &res_len, sizeof(res_len));			/* RES Length (bits, big endian) */
-		p += sizeof(res_len);
+		FR_DBUFF_CHECK_REMAINING_RETURN(&work_dbuff, vp->vp_length + 2);
```

Comment:
again, now redundant

Reply:
I will eliminate this one as well (and note when the corrected version is pushed); thanks.

Reply:
Also removed, and change pushed.